packages/donor/src/components/Wizard/WizardScreen.tsx

```diff
@@ -1,40 +1,103 @@
-import Stepper from "../basic/Stepper";
-import Button, { ButtonVariant } from "../basic/Button";
+import Button from "../basic/Button/ButtonV2";
+import Text from "../basic/Text";
+import styles from "./WizardScreen.module.scss";
+import { useState } from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import MobileStepper from "@material-ui/core/MobileStepper";
+import { BrowserRouter, Link } from "react-router-dom";
+import { PropTypes } from "@material-ui/core";
+import { ButtonVariant } from "../basic/Button";
 
-export interface WizardScreenProps {
-  imageUrl: string;
-  title: string;
-  content: string;
+const useStyles = makeStyles({
+  root: {
+    flexDirection: "column",
+    fontFamily: "Rubik",
+    backgroundColor: "#fff",
+  },
 
-  // Stepper props:
-  currentStep: number;
-  totalNumberOfSteps: number;
+  dot: {
+    margin: "8px",
+  },
 
-  // Action button
+  dots: {
+    margin: "30px 0",
+  },
+
+  dotActive: {
+    backgroundColor: "#A2D45E",
```

Comment:
What is this color? Should we use the green we have in other places?

Reply:
thats the color in the design..